exchanges/kucoin/kucoin_test.go

```diff
@@ -81,14 +79,16 @@ func TestMain(m *testing.M) {
 // Spot asset test cases starts from here
 func TestGetSymbols(t *testing.T) {
 	t.Parallel()
-	_, err := ku.GetSymbols(context.Background(), "")
+	symbols, err := ku.GetSymbols(context.Background(), "")
 	if err != nil {
 		t.Error("GetSymbols() error", err)
 	}
-	_, err = ku.GetSymbols(context.Background(), currency.BTC.String())
+	assert.NotEmpty(t, symbols, "should return all available spot/margin symbols")
+	symbols, err = ku.GetSymbols(context.Background(), "ETF")
 	if err != nil {
 		t.Error("GetSymbols() error", err)
 	}
+	assert.NotEmpty(t, symbols, "should return all available futures symbols")
```

Comment:
Did you have an issue where symbols weren't returned, but there was no error?

Reply:
Added commentary [here](https://github.com/thrasher-corp/gocryptotrader/pull/1369/commits/3c2ea05c069523acc3548970e26327e880f36c7c)

No it was just to test the usage, I thought originally it was the case.